store/tikv/txn.go

```diff
@@ -381,6 +376,14 @@ func (txn *tikvTxn) LockKeys(ctx context.Context, forUpdateTS uint64, keys ...kv
 		txn.committer.isFirstLock = len(txn.lockKeys) == 0 && len(keys1) == 1
 		err := txn.committer.pessimisticLockKeys(bo, keys1)
 		if err != nil {
+			wg := txn.asyncPessimisticRollback(ctx, keys1)
```

Comment:
If `asyncPessimisticRollback` fail, this transaction is still locking the key, while it doesn't think itself hold the lock...

Reply:
It doesn't affect the correctness.